tests/test_opera_rtc_s1.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+
+import hyp3_sdk
+import pytest
+import requests
+from osgeo import gdal
+
+from hyp3_testing import util
+from hyp3_testing.helpers import archive_tifs, job_tifs
+from hyp3_testing.opera_compare import compare_rtc_hdf5_files, compare_rtc_s1_products
+
+
+gdal.UseExceptions()
+CMR_URL = 'https://cmr.earthdata.nasa.gov/search/granules.umm_json'
+pytestmark = pytest.mark.golden
+
+
+@pytest.mark.nameskip
+def test_golden_submission(comparison_environments):
+    comparison_environments = [comparison_environments[1]]
```

Comment:
Is the intention to come back and update the tests when we make `OPERA_RTC_S1` available in prod? And is it worth leaving a TODO here?
```suggestion
    # TODO: remove this line when we're ready to run jobs in prod
    comparison_environments = [comparison_environments[1]]
```

Reply:
Didn't realize that we're never planning to run jobs in prod for this golden test.